README.md

```diff
@@ -2,18 +2,18 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML & CSS
+- Frameworks (none)
+- Technologies used (git & Github, Linters)
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://github.com/Progress2002/Portfolio)
 
 
 ## Getting Started
```

Comment:
- [ ] I would recommend you write how a user of your repo can get started by highlighting the steps it takes to get the local copy of the website to the user's local machine 😄